http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=655926

--- a/less.nro
+++ b/less.nro
@@ -27,10 +27,8 @@
 .I Less
 is a program similar to
 .IR more (1),
-but which allows backward movement
-in the file as well as forward movement.
-Also,
-.I less
+but it has many more features.
+.I Less
 does not have to read the entire input file before starting,
 so with large input files it starts up faster than text editors like
 .IR vi (1).
