http://bugs.debian.org/434417

--- a/main.c
+++ b/main.c
@@ -129,7 +129,7 @@
 	 * act like LESS_IS_MORE is set.
 	 */
 	s = last_component(progname);
-	if (strcmp(s, "more") == 0)
+	if (strcmp(s, "more") == 0 && isnullenv(lgetenv("LESS_IS_MORE")))
 		less_is_more = 1;
 
 	init_prompt();
--- a/opttbl.c
+++ b/opttbl.c
@@ -517,7 +517,7 @@
 	char *p;
 
 	p = lgetenv("LESS_IS_MORE");
-	if (!isnullenv(p))
+	if (!isnullenv(p) && p[0] == '1' && p[1] == '\0')
 		less_is_more = 1;
 
 	for (o = option;  o->oletter != '\0';  o++)
